frontend/src/pages/RequestorPage/RequestorPage.tsx

```diff
@@ -93,7 +93,7 @@ export const RequestorPage = () => {
     sessionHistory,
   );
 
-  const { mutate: makeRequest, isLoading: isRequestorRequesting } =
+  const { mutate: makeRequest, isPending: isRequestorRequesting } =
```

Comment:
I think we probably want to either use `isLoading` or `isPending` everywhere. Now `useRequestDetails` returns `isLoading`, while it's `isPending` here.

Reply:
i changed it to pending everywhere i could find so that we are aligned with react-query